### ISSUE TITLE
Parse has option to control casting ...

### ISSUE THREAD

**Issue by rufuspollock** [okfn/csv.js#4]
Should probably turn off auto-casting ...

**Comment by dwringer** [comment]
The current regexp check of "rxIsInt" is not sufficient to determine if a field represents an integer, since those larger than the largest supported integer by JS are truncated by the subsequent parseInt(). To demonstrate the problem, take a CSV row with a very long string composed of integers 0-9, parse it, serialize it, then parse it again.  The resulting value will be a different integer due to the truncation.

**Comment by rufuspollock** [comment]
@dwringer good point and thanks for sharing. I agree we should turn off auto-casting. If you - or someone else - ould like to do a PR that would be great 😉
